scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1373,6 +1384,11 @@ class Router(formatOps: FormatOps) {
               // Do whatever the user did if infix.
               Split(Space.orNL(newlines == 0), 0)
             )
+          else if (
+            style.trailingCommas.eq(TrailingCommas.keep) &&
```

Comment:
What about `TrailingCommas.always` shouldn't the behaviour be the same?

Reply:
then we need to add another label. in reality, what they all mean is, if there's a newline after the last param/arg, what should be done. none of them forced that newline, before this change.

Reply:
But currently with `TrailingCommas.always` we will remove the comma for anything shorter and add it for anything longer than a single line, which is unexpected. We should at least not remove the comma in that case.

Reply:
I am not even talking about when it's on the same line.

Let's say we have:
```scala
apply(a, b,
)
```

with `keep` we will get:
```
apply(
  a,
  b,
)
```

but with always we will get in rewritten to:

```
apply(a, b)
```

This changes if the line is longer so at the same moment another normal method might be rewritten from:
```scala
veryLongMethodName(a, b)
```

to 

```scala
veryLongMethodName(
  a,
  b,
)
```

This can even happen in the same file, two operations doing the apposite things.

Reply:
when i talked about "same line", i meant _after_ scalafmt chose the splits (not in the source code). if we want to force a trailing comma, we also need to force a newline, and i am going to add an explicit parameter for that.